### DISCUSSION TITLE
Final Topic Ideas

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#50] [Polls]
## Final Topic Ideas

Here are a couple of initial thoughts on my final project

- A Time and Project Management Website
- Personal Resume/Portfolio Site
- Microsite for a new product launch (fake product like GPS Running Shoes or something silly like that)
- New Digital Marketing Agency Website
- Company advertising corporate training classes
- Fan site for 1980 nostalgic toy collectors

**Comment by kassyoveson** [comment]
How cool are you and how did I miss this opinion poll? That was only the 2nd week in. Very impressive! Have you coded before?
